modules/app_aks/main.tf

```diff
@@ -56,3 +56,9 @@ resource "azurerm_role_assignment" "resource_group" {
   role_definition_name = "Reader"
   principal_id         = local.ingress_gateway_principal_id
 }
+
+resource "azurerm_role_assignment" "gateway" {
```

Comment:
huh, this was missing: https://github.com/wandb/terraform-azurerm-wandb/commit/72d2c882390a09c151392dc23251651081d1c22a